ambassador/ambassador/config.py

```diff
@@ -1313,6 +1317,40 @@ def tls_config_helper(self, name, amod, tmod):
 
         return some_enabled
 
+    def tracing_config_helper(self, name, amod):
+        tmp_config = SourcedDict(_from=amod)
+        tracing_config = amod['tracing']
+
+        # This is the name given to the cluster that 'host' points to
+        collector_cluster = "ambassador-tracing-service"
+        # For some  reason I'm not able to use this key value pair in the j2 template so it's hard-coded for now
+        tmp_config["collector_cluster"] = collector_cluster
+
+        for key in tracing_config.keys():
+            if key.startswith('_'):
+                continue
+
+            if key == 'driver':
+                driver = tracing_config.get(key, '').lower()
+                # Only allow user to create a 'zipkin' or 'lightstep' driver for now. Update with envoy
+                assert driver in ['zipkin', 'lightstep']
```

Comment:
We don't use `assert` for error handling anywhere else; better to use `post_error` instead. See `Config.add_clusters_for_mapping` and `Config.auth_helper` for examples.